packages/mwp-app-render/src/components/SEOHead.jsx

```diff
@@ -42,13 +44,13 @@ export const SEOHeadComponent = ({
 	robots,
 	route,
 }) => {
-
 	const metaData = generateMetaData({
 		appPath: `meetup:/${route}`,
 		baseUrl,
 		pageDescription,
 		imageUrl,
 		localeCode,
+		groupLocaleCode,
```

Comment:
I couldn't find where signature of this method changed. Do we need to pass `groupLocaleCode` here?